schema.sql

```diff
@@ -1,13 +1,50 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
+CREATE TABLE Animals (
     id            INT GENERATED BY DEFAULT AS IDENTITY,
     name TEXT NOT NULL, 
     date_of_birth DATE NOT NULL, 
     escape_attempts INT NOT NULL, 
     neutered BOOLEAN NOT NULL, 
     weight_kg DECIMAL NOT NULL,
     PRIMARY KEY(id)
-);
+)
+
+ALter table Animals ADD species TEXT NOT NULL;
+
+CREATE TABLE owners (
+    id            SERIAL PRIMARY KEY,
+    full_name TEXT NOT NULL;
+    age INT NOT NULL; 
+)
+
+CREATE TABLE species (
+    id        SERIAL PRIMARY KEY,
+   name TEXT NOT NULL; 
```

Comment:
- [ ] kindly user comas `,` at the end of the line in the `()`, note that `; ` are user at the end of the full instruction(query). Hint example :point_down: 

```sql
CREATE TABLE owners (
    id            SERIAL PRIMARY KEY,
    full_name TEXT NOT NULL;
    age INT NOT NULL; 
)
```
this one :point_down:  is better and does not throw errors 

```sql

CREATE TABLE owners (
    id            SERIAL PRIMARY KEY,
    full_name TEXT NOT NULL,
    age INT NOT NULL,
);
```

Reply:
Thanks for pointing out the syntax error. Updated!